### PR TITLE
Updated Quickstart link

### LINKED ISSUE #2962: Wrong link in the README.md
## Documentation version
3.23

## Incorrect section
README.md

## Description of the problem
In https://github.com/syslog-ng/syslog-ng#installation-from-source, there is a link to 
https://www.balabit.com/sites/default/files/documents/syslog-ng-ose-latest-guides/en/syslog-ng-ose-guide-admin/html/compiling-syslog-ng.html but it leads to https://www.oneidentity.com/ which is useless.

## Proposed solution
The link should be https://www.syslog-ng.com/technical-documents/doc/syslog-ng-open-source-edition/3.23/administration-guide/10#TOPIC-1268421

### PR DESCRIPTION
**Pull request by clemsciences**
#2962 There is actually two ways to correct the README.md:

- change the quickstart quide link to https://www.syslog-ng.com/technical-documents/list/syslog-ng-open-source-edition/ so that this link is correct for all future versions (by the way, it would be good to rename quickguide to technical documentation),
- change the quickstart guide link to https://www.syslog-ng.com/technical-documents/doc/syslog-ng-open-source-edition/3.23/administration-guide/11#TOPIC-1268425 so that it is directly linked to what is expected, however, this must be changed for every new version.

Resolves #2962

### CODE REVIEW

**Comment by kira-syslogng** [comment]
This user does not have permission to start the build. Can one of the admins verify this patch and start the build?
(admin: you have the next options (make sure you checked the code):
"ok to test" to accept this pull request (and further changes) for testing
"test this please" for a one time test run
"add to whitelist" add author of a Pull Request to whitelist (globally, be careful, it means this user can trigger kira for any PR)
do nothing -> CI won't start)

**Comment by kira-syslogng** [comment]
This user does not have permission to start the build. Can one of the admins verify this patch and start the build?
(admin: you have the next options (make sure you checked the code):
"ok to test" to accept this pull request (and further changes) for testing
"test this please" for a one time test run
"add to whitelist" add author of a Pull Request to whitelist (globally, be careful, it means this user can trigger kira for any PR)
do nothing -> CI won't start)

**Comment by kira-syslogng** [comment]
This user does not have permission to start the build. Can one of the admins verify this patch and start the build?
(admin: you have the next options (make sure you checked the code):
"ok to test" to accept this pull request (and further changes) for testing
"test this please" for a one time test run
"add to whitelist" add author of a Pull Request to whitelist (globally, be careful, it means this user can trigger kira for any PR)
do nothing -> CI won't start)

**Comment by szemere** [comment]
ok to test

**Comment by szemere** [comment]
Hello,

please `git commit --sign-off` your commit.

Thank You

**Comment by MrAnno** [comment]
Please also add a prefix to the commit message that specified the "module" or "unit" you modified, for example:

```
README: updated Quickstart link
or
docs: Updated Quickstart link
etc.
```

This with the signoff will pass our commit check in Travis.

File: README.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ In which case the resulting message will be:
 ```
 name1=value1 name2=value2
 ```
-For a brief introduction to configuring the syslog-ng application, see the [quickstart guide](https://syslog-ng.com/documents/html/syslog-ng-ose-latest-guides/en/syslog-ng-ose-guide-admin/html/chapter-quickstart.html).
+For a brief introduction to configuring the syslog-ng application, see the [quickstart guide](https://www.syslog-ng.com/technical-documents/list/syslog-ng-open-source-edition/).
```
Comment (MrAnno) [inline README.md]: I could not find a link to the non-versioned latest guide, which could be linked directly to the quickstart section.
@fekete-robert Do you know about such link?

We should probably rephrase this sentence if we can't provide a link directly to the quickstart guide.

**Comment by fekete-robert** [comment]
Hi,
This link should always point to the latest adminguide:
https://support.oneidentity.com/technical-documents/syslog-ng-open-source-edition/administration-guide/
And this one to the quickstart chapter:
https://support.oneidentity.com/technical-documents/syslog-ng-open-source-edition/administration-guide/the-syslog-ng-ose-quick-start-guide

File: README.md (hunk 1 above)
Comment (MrAnno) [reply README.md]: @fekete-robert Thank you!

**Comment by szemere** [comment]
Hello @clemsciences,
sorry for bothering you this much, but I think there were a small mistake with your last git push.

Looks like this commit was the good one https://github.com/syslog-ng/syslog-ng/commit/595e8ec2deef2249967f99d1dc293eea8d1d152d and somehow it got overwritten.

Br,
Laci

**Comment by gaborznagy** [comment]
It seems like I'm making a [bicycle-shed-effect](https://en.wikipedia.org/wiki/Law_of_triviality) out of this, but it really is just a review.

I think this was your original finding:
`[syslog-ng compiling instructions](https://www.balabit.com/sites/default/files/documents/syslog-ng-ose-latest-guides/en/syslog-ng-ose-guide-admin/html/compiling-syslog-ng.html`
this should changed according to @fekete-robert to
`https://support.oneidentity.com/technical-documents/syslog-ng-open-source-edition/administration-guide/installing-syslog-ng`

If you had enough, I can send a patch onto your fork, so it will be included in this PR.

Thanks for reporting and fixing these bugs! :+1:

**Comment by gaborznagy** [comment]
@clemsciences fair enough, we can fix the other broken link.

Thanks for finding the broken links.

**Comment by clemsciences** [comment]
@gaborznagy, oh you mean you found an other broken link, I can correct if you want, sorry, I misunderstood your comment.

**Pull request merged**